src/cycax/cycad/engines/cycax_part_freecad.py

```diff
@@ -233,7 +233,7 @@ def build(self, in_name: Path):
 
         filepath = f"{self._base_path}/{name}/{name}"
```

Comment:
I wonder if we should not make a file name creation method as part of the base class.

e.g. (untested pseudo code) 
```
def get_file(self, postfix:str = None, extension: str):
       filepath = self._base_path / self.name
       if postfix is None:
           filename = f"{self.name}{self.engine_file_postfix}.{extension}"
       else: 
           filename = f"{self.name}{postfix}{self.engine_file_postfix}.{extension}"
       return filepath / filename
```

self.engine_file_postfix - Is a class level variable  (attribute) that holds a postfix added to all files created with the class.
self.postfix - Is a postfix added to files, if a method makes multiple files of the same type, this is needed. Say many png files for different sides and perspectives. We need to do this with FreeCAD and Blender.